apps/google-analytics-4/frontend/src/helpers/handle-date-range/handle-date-range.ts

```diff
@@ -0,0 +1,31 @@
+import { DateRangeType } from 'types';
```

Comment:
another nitty file naming complaint - i feel like the term `handle` should be reserved for function naming. This feels more like a DateRangeUtils or DateRangeHelpers type of file. Also, I think Gary is pushing for camel casing file names and not using dashes.